File: src/cpp/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
            string s = to_string(x);
            int start = 0;
            int end = s.length() - 1;

            while(start < end)
            {
                if(s[start++] != s[end--])
                    return false;
            }

            return true;
        }
};

// Submission- https://leetcode.com/submissions/detail/814241551/

File: src/python/palindrome-number.py
#https://leetcode.com/problems/palindrome-number/

class Solution:
    def isPalindrome(self, x: int) -> bool:
        y = str(x)
        lst = list(y)
        lst.reverse()
        y = "".join(lst)
        try:
            y = int(y)
        except:
            ValueError
        if x == y:
            return True
        else:
            return False